conans/test/integration/command_v2/list_test.py

```diff
@@ -747,3 +746,27 @@ def test_list_html_custom(self):
         c.save({"list_packages.html": '{{ base_template_path }}'}, path=template_folder)
         c.run("list lib/0.1#latest --format=html")
         assert template_folder in c.stdout
+
+
+class TestListCompact:
+    def test_list_compact(self):
+        c = TestClient()
+        c.save({"conanfile.py": GenConanfile("pkg", "1.0").with_settings("os", "arch")
```

Comment:
I think we might want to exercise the codepath of no options and showing the timestamps, just in case :)

Reply:
Done